iothub_client/tests/iothubtr_amqp_tel_msgr_ut/iothubtr_amqp_tel_msgr_ut.c

```diff
@@ -1665,11 +1665,109 @@ static TELEMETRY_MESSENGER_HANDLE create_and_start_messenger2(TELEMETRY_MESSENGE
     return handle;
 }
 
+char* umock_stringify_BINARY_DATA()
+{
+    char* result = (char*)TEST_malloc(1);
+    result[0] = '\0';
+    return result;
+}
+
+int umock_are_equal_BINARY_DATA()
+{
+    //force fall through to success bypassing access violation
+    int result = 1;
+    return result;
+}
+
+int umock_copy_BINARY_DATA()
+{
+    //force fall through to success bypassing access violation
+    int result = 0;
+    return result;
+}
+
+void umock_free_BINARY_DATA()
+{
+    //do nothing
+}
+
+char* umock_stringify_TELEMETRY_MESSENGER_MESSAGE_DISPOSITION_INFO(const TELEMETRY_MESSENGER_MESSAGE_DISPOSITION_INFO* value)
+{
+    char* result = (char*)TEST_malloc(1);
+    (void)value;
+    result[0] = '\0';
+    return result;
+}
+
+int umock_are_equal_TELEMETRY_MESSENGER_MESSAGE_DISPOSITION_INFO(const TELEMETRY_MESSENGER_MESSAGE_DISPOSITION_INFO* left, const TELEMETRY_MESSENGER_MESSAGE_DISPOSITION_INFO* right)
+{
+    int result;
+
+    if (((left == NULL) && (right != NULL)) || ((right == NULL) && (left != NULL)))
+    {
+        result = 0;
+    }
+    else
+    {
+        if ((left->source != right->source) || (left->message_id != right->message_id))
```

Comment:
Same here. Possible SEGFAULT when both left and right are NULL

Reply:
fixed